jsonrpc/jsonrpc.go

```diff
@@ -91,6 +96,24 @@ func (j *JSONRPC) setupHTTP() error {
 	return nil
 }
 
+// The middlewareFactory builds a middleware which enables CORS using the provided config.
+func middlewareFactory(config *Config) func(http.Handler) http.Handler {
+	middleware := func(next http.Handler) http.Handler {
```

Comment:
You can inline this return

Reply:
Fixed in https://github.com/0xPolygon/polygon-edge/pull/368/commits/91a80b1f9db44a76c758f6c7b04c9a5bc9c2c793